testing/tests/DevExpress.ui/collectionWidgetItem.tests.js

```diff
@@ -151,3 +151,13 @@ QUnit.test('disabled should be rendered correctly after value changed', function
 
     assert.ok($item.hasClass(DISABLED_STATE_CLASS));
 });
+
+QUnit.test('aria-disabled attribute should be set if the item\'s disabled prop is true', function(assert) {
+    const widget = new CollectionWidget($('<div>'), {
+        items: [{ disabled: true }]
+    });
+
+    const $item = widget.itemElements().eq(0);
+
+    assert.ok($item.attr('aria-disabled'), 'element has aria-disabled attribute');
+});
```

Comment:
Don't we want to test this case for a `disabled` property runtime change?

Reply:
Okay, let's do it! I added a new test in this commit [25bf9f775185867becc632f8384161cc5d472868].